src/components/header/header.css

```diff
@@ -1,14 +1,14 @@
 .header {
-  width: 100vw;
+  width: 100%;
```

Comment:
Siguiendo mi ultimo comentario, donde se agrego el gap al `header-left` y `header-right`, hay algo que no me gusta y es que la opcion "Listado de alumnos" queda pegado al borde izquierdo, por lo que le agregaria un padding horizontal al header. Lo estuve probando y me gusta como queda, pero al hacer el header con `width: 100%` este se sale de la pantalla, por lo que haria lo siguiente


```suggestion
  padding: 0px 15px;
```

<img width="2073" height="96" alt="Image" src="https://github.com/user-attachments/assets/00e481c9-8c6b-4c8e-8b85-959b153699e8" />